packages/react-html/src/tests/context.test.tsx

```diff
@@ -92,9 +92,9 @@ describe('<Provider />', () => {
     const manager = new Manager();
 
     mount(
-      <Provider manager={manager}>
+      <HtmlProvider manager={manager}>
         <div />
-      </Provider>,
+      </HtmlProvider>,
```

Comment:
These are I assume just problems with the base branch eh?

Reply:
Yes. I was working on tests.